src/utils/processInput.ts

```diff
@@ -48,9 +48,9 @@ export const sanitizeRow = ({
         collection: other_collections,
       }
     ),
-    query: (artist_ids || gene_ids || tag_id || keyword) && {
-      artist_ids: splitmap(artist_ids),
-      gene_ids: splitmap(gene_ids),
+    query: {
```

Comment:
Previously, this was creating a null `query` property on collections if they don't have at least one of artist_ids, gene_ids, tag_ids or keywords. This was the direct cause of the current problem where collections without queries were breaking ingestion.